database/seeds/DatabaseSeeder.php

```diff
@@ -15,6 +15,7 @@ public function run(): void
         $this->call(BuslineSeeder::class);
         $this->call(StationSeeder::class);
         $this->call(GroupSeeder::class);
-        $this->call(BuslineStationSeeder::class);
+        $this->call(PathSeeder::class);
```

Comment:
It's a good moment to think about what would happen if these seeders would be called on the production environment. Maybe it would be a good idea to ensure that no stupid data would land on prod database by the accident.